rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2022-05-23
+nightly-2022-08-12
```

Comment:
Maybe something like that would be more `pro`? It then automatically installs the wasm target.
```
[toolchain]
channel = "nightly-2022-08-12"
targets = [ "wasm32-unknown-unknown" ]
```

Reply:
I wrestled with it in a past, but the actions-rs/toolchain action does not understand toml and it's still in v1. Probably unmaintained too: https://github.com/actions-rs/toolchain/issues/216

Reply:
Ok, fine, then you can ignore my comment.